src/components/Write/WriteEvaluation.jsx

```diff
@@ -92,17 +50,17 @@ const WriteEvaluation = ({ setModalIsOpen, row, type }) => {
           />
         </MobileContent>
         {EvalutionSliderOptions.map(({ id, name }) => {
-          const [value, Slider] = SliderOptions[id];
+          const { state, Slider } = SliderOptions[id];
           return (
             <Fragment key={id}>
               <Content>
                 <ContentTitle>{name}</ContentTitle>
-                <Slider /> <Score>{value}</Score>
+                <Slider /> <Score>{state}</Score>
```

Comment:
이부분 프리티어 적용된건지 확인해봐야할거같은데 영진님 아직 프리티어 적용이 잘안되어있으신건지 확인부탁드려요!

```suggestion
                <Slider /> 
                <Score>{state}</Score>
```

이렇게 되어야할거같아서요.

Reply:
엇 프리티어는 잘 적용이 되는데, 확인해보니 두 경우 모두 적용이 되는군요... 일단 수정하겠습니다!
![image](https://user-images.githubusercontent.com/83450991/220224349-ebf3e0b6-86b3-41ff-9268-5b2146ccba73.png)

Reply:
두 경우가 전부 되면 안 될텐데요. 프로젝트에 심어놔야겠어요 ㅎㅎ 감사합니다 덕분에 찾았어요